### PR TITLE
076 Project Page Component

### LINKED ISSUE #76: Project Page Component
## Description

Fix styling of project page component as shown in Figma.

## Acceptance Criteria

- [ ] Component matches Figma design.

## Definition of Done

- [ ] Acceptance criteria met
- [ ] PR approved by 2 others
- [ ] Merged into the main

### PR DESCRIPTION
**Pull request by panmann7**
## Context
Matching the project page design to what is shown on Figma. 

Closes #76 

## What Changed?
Made small changes to fix the CSS styling of the project page component. 

## How To Review
To view, go to next\app\projects\page.tsx

### CODE REVIEW

**Review by Oculux314 — CHANGES_REQUESTED** [review]
Nice, like the alt text addition 👍 

Something I remember Yvonne wanted was having all the text line up towards the middle like this:
image
(at the moment the text is crushed to the right if there isn't enough of it)
image

Also! Might be impossible, just msg here if so, but would it be possible to make this text extrabold? (You may need to edit tailwind.config.js)
image

File: next/app/projects/page.tsx [hunk 1]
```diff
@@ -21,15 +21,15 @@ export default async function ProjectsPage() {
       <section className="h-[85vh] flex flex-col items-center justify-center relative">
         <Image
           src={getLargestImageUrl(projectsPage.BackgroundImage)}
-          alt=""
-          className="absolute inset-0 -z-10 object-cover w-full h-full brightness-50"
+          alt="Projects page background image"
+          className="absolute inset-0 -z-10 object-cover object-[center_bottom] w-full h-full brightness-50"
```
Comment (Oculux314) [inline next/app/projects/page.tsx]: P.s.! Is there a reason you chose to bottom-align images? Not right or wrong just curious~~

**Review by YvonneLiew — CHANGES_REQUESTED** [review]
Yeah I agree with what Nate said above with regards to the centering of the text. Can you make it so that if there's little text, it's more centred?

**Comment by Oculux314** [comment]
@panmann7 Sorry this is great work but since the Projects page is being redone I'm going to close this PR! Look forward to seeing your work with the projects images :D

**Pull request closed without merging**